src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,13 +189,20 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    string val = "NoProcessFoundForGivenName";
+                    string errorText = ProcessResources.NoProcessFoundForGivenName;
+                    if(int.TryParse(pattern, out int x) && x>0)
+                    {
+                        errorText = ProcessResources.RecommendIdTagForGivenName;
+                        val = "RecommendIdTagForGivenName";
+                    }
```

Comment:
Add a newline at the end of this if for readability.
```suggestion
                    }

```